web_src/js/standalone/devtest.js

```diff
@@ -1,11 +1,30 @@
+import '../bootstrap.js';
+import '../jquery.js';
+import '../../fomantic/build/semantic.js';
+import $ from 'jquery';
```

Comment:
Another concern, does this "Fomantic" work with the code in "index.js"?

For example, we have our modal/dropdown/checkbox code in "index.js". If "devtest" is a separate module, then I guess all code here are independent of the code in "index.js", then a lot of mechanisms in "index.js" won't work with the code here?

Reply:
I removed the imports, they are all duplicate and we should not do that.